docs/source/model-registry.rst

```diff
@@ -732,3 +736,84 @@ At a later point, if that archived model is not needed, you can delete it.
         name="sk-learn-random-forest-reg-model", version=3, stage="Archived"
     )
 
+.. _migrating-from-stages:
+
+Migrating from Stages
+=====================
+
+As of MLflow 2.9.0, Model Stages have been deprecated and will be removed in a future major release. This is the culmination of extensive feedback on the inflexibility of model stages for expressing MLOps workflows, from which we developed and introduced of new tools for managing and deploying models in the MLflow Model Registry. Learn more below.
+
+New model deployment tools
+--------------------------
+
+Model stages were used to express the lifecycle of MLflow Models for productionization and deployment. Users transitioned model versions through four fixed stages (from **none**, to **staging**, to **production**, and then to **archived**) as they proposed, validated, deployed, and deprecated models for their ML use-cases. In doing so, model registry stages provided labeling and aliasing functionality for the model versions, by denoting the status of a model version in the UI and providing named references to model versions in the code (e.g. ``/Staging`` in the model URI). Model registry stages were also used to denote the environment that the model is in, though it was not possible to set up access controls for them.
+
+To replace and improve upon stages, we elevated **model version tags** in the UI and introduced **model version aliases** to provide flexible and powerful ways to label and alias MLflow models in the Model Registry. We also made it possible to **set up separate environments** for your models and configure access controls for each environment.
+
+**Model version tags**
+
+Model version tags can be used to annotate model versions with their status. For example, you could apply a tag of key ``validation_status`` and value ``pending`` to a model version while it is being validated and then update the tag value to ``passed`` when it has passed smoke tests and performance tests.
+
+**Model version aliases**
+
+Model version aliases provide a flexible way to create named references for particular model versions, and are useful for identifying which model version(s) are deployed within an environment. For example, setting a **champion** alias on a model version enables you to fetch the model version by that alias via the :meth:`~mlflow.client.MlflowClient.get_model_version_by_alias` client API or the model URI ``models:/<registered model name>@champion``. Aliases can be reassigned to new model versions via the UI and client API. Unlike model registry stages, more than one alias can be applied to any given model version, allowing for easier A/B testing and model rollout.
+
+**[New] Set up separate environments for models**
```

Comment:
```suggestion
**Set up separate environments for models**
```

Should we remove the "new" block? Or have some issue etc to track removing it? We already mention "New deployment tools" above and this will become not-new in 2.10.0, so not sure it makes sense to keep here. Creating & ACLing separate registered models is also possible in older MLflow versions

Reply:
Makes sense!